core/endpoint_test.go

```diff
@@ -23,6 +23,69 @@ func TestEndpoint_IsEnabled(t *testing.T) {
 	}
 }
 
+func TestEndpoint_Type(t *testing.T) {
+	type fields struct {
+		URL string
+		DNS *DNS
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   EndpointType
+	}{{
+		name: "",
```

Comment:
Any chance you can give these tests a proper name? (i.e. `dns`, `tcp`, `icmp`, etc.)

Actually, you could probably just remove the name altogether, and use `string(tt.want)`

Reply:
hi @TwiN 
change applied